dashboard/README.md

```diff
@@ -1,15 +1,131 @@
 # Games Tracker Dashboard
 
+This dashboard is designed to provide visualisations and insights into game releases from popular platforms such as Steam, GOG, and Epic Games. Users can track game releases, subscribe to updates, and explore trends based on operating systems, genres, and more.
+
+The dashboard is geared toward:
+
+- **Game enthusiasts**, who can use it to have all PC game releases in one place and track new releases easily.
+- **Industry professionals**, who can use the insights for analysing trends and understanding the gaming market.
+
+The dashboard is built using Streamlit, allowing for interactive and responsive visualisations that help users explore trends across platforms.
+
+## Features
+
+- Visualise game releases across platforms with customizable filters.
+- Subscribe to weekly reports and genre-specific game updates via email.
+- Explore genre popularity, operating system trends, and more.
+
 ## Wireframe
 
-1. Dashboard (graphs that will be customised depending on user filters)
+1. **Dashboard** (graphs that will be customised depending on user filters)
 
 ![Dashboard](wireframe_dashboard.png)
 
-2. About (information about the project and dashboard)
+2. **About** (information about the project and dashboard)
 
 ![About](wireframe_about.png)
 
-3. Subscribe (information about subscribing to the mailing list)
+3. **Games** (list of games with search functionality)
+
+![Games](wireframe_games.png)
```

Comment:
Do we still need to be using wireframes, could we not use real screenshots in the README? since it's actually developed now.